jsettlers.graphics/res/layout/WarriorsLayout.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE element SYSTEM "layout.dtd">
+<element x="9" y="219" maxx="127" maxy="435">
+	<element class="Label" x="9" maxx="127" y="219" maxy="234">
+		<localized label="warriors_title" />
+		<font-size size="NORMAL" />
+	</element>
+
+
+	<element class="Label" x="9" maxx="127" y="235" maxy="250">
+		<localized label="upgrade_warriros_progress" />
+		<font-size size="NORMAL" />
+	</element>
```

Comment:
I read your comment too late and in the meantime I introduced an UpgradeProgressLabel.
Which way is the preferred one?